libs/providers/env-var/project.json

```diff
@@ -50,7 +49,7 @@
         "compiler": "tsc",
         "generateExportsField": true,
         "umdName": "Env Var",
-        "external": ["typescript"],
+        "external": "all",
```

Comment:
There was a change in nx:rollup that caused anything not specified here to auto-bundle (which admittedly makes sense). That's not what we want here, or at least not what we've been doing. The bundling also caused a bunch of breakage in flagd, some of the gRPC code didn't bundle well.

I create a doc PR in nx: https://github.com/nrwl/nx/pull/17005